src/assets/AccountTrackerController.ts

```diff
@@ -42,7 +42,7 @@ export class AccountTrackerController extends BaseController<
   AccountTrackerConfig,
   AccountTrackerState
 > {
-  private ethQuery: any;
+  private ethQuery: EthQuery | null;
```

Comment:
This property can be `null` because it is not set in the constructor (or at least I didn't see that it was set in the constructor).